README.md

```diff
@@ -606,6 +544,83 @@ Web Development, React, Node.js, CSS, Next.js, Wordpress
 - [GitHub](https://github.com/hawkins)
 - [Website](https://hawkins.is)
 
+### KensoDev
```

Comment:
Ha! Thanks for fixing my inability to know the alphabet and mix J and K. I can't believe I did that. 🤦‍♂️